elearning-backend/app/Http/Controllers/API/AdminUsersController.php

```diff
@@ -0,0 +1,75 @@
+<?php
+
+namespace App\Http\Controllers\API;
+
+use App\Http\Controllers\Controller;
+use App\Http\Requests\AdminUsersPostRequest;
+use App\Models\User;
+use App\Models\UserProfile;
+use Illuminate\Http\Request;
+use Illuminate\Support\Facades\File;
+use Laravel\Sanctum\PersonalAccessToken;
+
+class AdminUsersController extends Controller
+{
+    public function index(Request $request)
+    {
+        // Exculde current user
+        $currentUser = PersonalAccessToken::findToken($request->input('token'))->tokenable()->with('profile')->get()->first();
+        $users = User::where("id", "!=", $currentUser->id)->with('profile')->get();
+
+        return response()->json([
+            'users'=> $users,
+        ]);
+    }
+
+    public function store(Request $request)
+    {
+        $user = User::where("id", $request->input('user'))->with('profile')->get()->first();
+
+        return response()->json([
+            'user'=> $user,
+        ]);
+    }
+
+    public function update(AdminUsersPostRequest $request)
+    {
+        $user = User::where("id", $request->input("user"))->with('profile')->get()->first();
```

Comment:
Dont pass the user id.
Instead of `$request->input("user"))`
You can utilize the sanctum middleware, below are the examples:
```
Auth::user()->id
auth()->user()->id
auth('sanctum')->user()->id
```